Tools/WinMLRunner/Main.cpp

```diff
@@ -322,7 +332,7 @@ HRESULT EvaluateModels(
 
         try
         {
-            model = LoadModel(path, args.PerfCapture() || args.PerIterCapture(), output, args, 0);
+            model = LoadModel(path, args.PerfCapture() || args.SaveTensor() || args.PerIterCapture(), output, args, 0);
```

Comment:
Why do we want to capture load model performance if we only want to save tensor?